CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to Presto
+
+Presto is an Apache 2.0-licensed open source distributed SQL query engine for running interactive analytic queries against data sources of all sizes, ranging from gigabytes to petabytes.
+
+Presto was designed and written from the ground up for interactive analytics and approaches the speed of commercial data warehouses while scaling to the size of organizations like Facebook.
+
+The community owned and driven Presto project is supported by the [Presto Foundation](https://prestodb.io), an independent nonprofit organization with open and neutral governance, hosted under the [Linux Foundation](https://linuxfoundation.org). Learn more about the [Presto Foundation](https://prestodb.io) here.
+
+All Presto development occurs on [GitHub](https://github.com/prestodb), following open source best practices.
+
+## Technical governance
+
+Presto follows industry best practices, with the technical project and its governance distinctly separate from the Presto Foundation.  The [Technical Charter](https://github.com/prestodb/tsc/blob/master/CHARTER.md) establishes the Presto Technical Steering Committee as the body which oversees all technical aspects of the open source project.  By contrast, the [Presto Foundation](https://prestodb.io) Governing Board is responsible for raising and distributing funds to support project activities, and setting policy.  While the two groups are separate, they work in close coordination to establish, guide, and support activities that lead to a long-term, sustainable future for Presto.
+
+### Project roles and responsibilities
+
+#### Contributors
+
+Contributors are any members of the open source community who participate in project develompent.  Contributors may be participation on their own behalf or on behalf of their employer.  Membership in the [Presto Foundation](https://prestodb.io) is not required in order to contribute, although contributors must have signed a CLA prior to their code being merged (see below).
+
+#### Committers
+
+Committers are members of the contributor community who have demonstrated an ongoing commitment to the health and sustainability of the Presto project through good citizenship and wise stewarship.  These individual have been elevated to a decision-making role, and responsible for reviewing and merging code from contributors.  They may also be involved in the release process or other critical project roles.
```

Comment:
typo in `stewarship`

Reply:
`These individual` -> `These individuals`

Reply:
Thanks, I've fixed these in another patch.